docs/source/install/install.rst

```diff
@@ -360,20 +315,19 @@ you can install MathJax locally.
 
 A quick and easy method is to install it from a python session::
 
-    from IPython.external.mathjax import install_mathjax
-    install_mathjax()
+    python -m IPython.external.mathjax
 
 If you need tighter configuration control, you can download your own copy
 of MathJax from http://www.mathjax.org/download/ - use the MathJax-2.0 link.
 When you have the file stored locally, install it with::
 
-	python -m IPython.external.mathjax /path/to/source/mathjax-MathJax-v2.0-20-g07669ac.zip
+    python -m IPython.external.mathjax /path/to/source/mathjax-MathJax-v2.0-20-g07669ac.zip
 
 For unusual needs, IPython can tell you what directory it wants to find MathJax in::
 
-	python -m IPython.external.mathjax -d /some/other/mathjax
+    python -m IPython.external.mathjax -d /some/other/mathjax
 
-By default Mathjax will be installed in your ipython profile directory, but you
+By default Mathjax will be installed in your ipython directory, but you
 can make system wide install, please refer to the documentation and helper function 
```

Comment:
this sentence doesn't read right - "you can make system wide install". Not your fault, but while we're working on this, may as well clean it up.